src/tm_mad/ceph/postmigrate

```diff
@@ -36,6 +36,21 @@ TEMPLATE_64=$6
 
 #--------------------------------------------------------------------------------
 
+# Get custom datastore values
+DRIVER_PATH=$(dirname "$0")
+XPATH="${DRIVER_PATH}/../../datastore/xpath.rb --stdin"
+
+unset XPATH_ELEMENTS i j
+
+while IFS= read -r -d '' element; do
+  XPATH_ELEMENTS[i++]="$element"
+done < <(onedatastore show -x "$DSID" | $XPATH \
+  /DATASTORE/TEMPLATE/SHARED)
+
+SHARED="${XPATH_ELEMENTS[j++]:-$SHARED}"
```

Comment:
1) Perhaps I not so understood purpose of this variable, so it is possible to rename to other name, but I consider that this option is necessary for such cases.
Any ideas about it?
2) Fixed